docs/spec/README.md

```diff
@@ -23,10 +23,7 @@ block.
 - [Mint](./mint) - Staking token provision creation.
 - [Params](./params) - Globally available parameter store.
 - [Supply](./supply) - Total supply of the chain.
-
-## Interchain standards
-
-- [ICS30](./_ics/ics-030-signed-messages.md) - Signed messages standard.
```

Comment:
Should this have been removed?

Reply:
yeah, all standards are now on the ICS repo